src/ResourceManager/Network/Commands.Network.Test/NrpTeamAlias.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Commands.Network.Test
+{
+    class NrpTeamAlias
+    {
+        // Below is the list of aliases to contact on test behavior
+
+        // Virtual Appliance dev team
+        public const string nvadev = "nvadev";
+
+        // SDN NRP Dev Team
+        public const string sdnnrp = "sdnnrp";
+
+        // Pankaj's Team
+        public const string pgtm = "pgtm";
+
+        // Windows Azure SLB Dev Team
+        public const string slbdev = "slbdev";
+
+        // Brooklyn FTEs
+        // Split into subsets due to tests' long running time
+        public const string brooklynft = "brooklynft";
+        public const string brooklynft_subset1 = "brooklynft_subset1";
```

Comment:
@number213 how were the tests split into subsets? It would be nice to document that here so the next person to add a test to the Brooklyn test suite knows which subset to place it into (or if they should create a new subset)

Reply:
I think it's related to a code path they add tests, but deifnetelly we need a comment here

Reply:
Added comments